_posts/2018-08-14-utah-sgid-statewide-roads-data-layer-updates-08142018.md

```diff
@@ -10,34 +10,33 @@ tags:
   - roads
 ---
 
-Updates were recently made to the SGID10.Transportation.Roads feature class that resides on the [Utah SGID ArcSDE database server]({{ "/sgid-database/" | prepend: site.baseurl }}).
-
-Updated roads data is also available as shapefiles and file geodatabase files for download on the [SGID Roads and Highway System download page]({{ "/data/transportation/roads-system/" | prepend: site.baseurl }}).
-
-These updates are also reflected in [AGRC's address locators]({{ "/data/address-geocoders-locators/" | prepend: site.baseurl }}) and the [Utah Next-Generation 911 GIS Database]({{ "/data/911/" | prepend: site.baseurl }}).
+Updates were recently made to the SGID10.Transportation.Roads feature class that resides on the [Utah SGID ArcSDE database server]({{ "/sgid-database/" | prepend: site.baseurl }}). Updated roads data is also available as shapefiles and geodatabase files for download on the [SGID Roads and Highway System download page]({{ "/data/transportation/roads-system/" | prepend: site.baseurl }}). These updates are also reflected in [AGRC's address locators]({{ "/data/address-geocoders-locators/" | prepend: site.baseurl }}) and the [Utah Next-Generation 911 GIS Database]({{ "/data/911/" | prepend: site.baseurl }}).
 
 
 The following are highlights from this month's update:
 
-#### County Updates: Beaver
+#### County Updates 
+New roads were added and road names and address ranges were updated for the following counties:
```

Comment:
Since we just listed the same text over and over for each of these bullet points, I thought it'd make more sense to just list it at the top. Does this work?